modules/iam-profile/outputs.tf

```diff
@@ -0,0 +1,4 @@
+output "aws_iam_role_arn" {
+  description = "aws iam role arn"
+  value       = aws_iam_role.role.arn
```

Comment:
We need "integration_name".
API can re-compute the the ARN for each account.
comments?

Reply:
yes we can re-compute ARN , because that would have to be done anyways with the account ID's of all the child accounts